scheduled-jobs/build/bug-sweeps/sweep/Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '90'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    commonlib.ocpVersionParam('BUILD_VERSION'),
+                ]
+            ]
+        ]
+    )
+
+    echo "Initializing bug sweep for ${params.BUILD_VERSION}. Sync: #${currentBuild.number}"
+
+    currentBuild.displayName = "${params.BUILD_VERSION} bug sweep"
+
+    stage("Version dumps") {
+        buildlib.elliott "--version"
+        sh "which elliott"
+    }
+
+    stage("kinit") {
+	buildlib.kinit()
+    }
+
+    try {
+	stage("Sweep bugs") {
+	    currentBuild.description = "Searching for and attaching bugs"
+	    buildlib.elliott "--group=openshift-${params.BUILD_VERSION} find-bugs --auto --use-default-advisory rpm"
+	    currentBuild.description = "Ran bug attaching command without errors"
+	}
+    } catch (findBugsError) {
+	currentBuild.description = "Error sweeping:\n${findBugsError}"
```

Comment:
How about re-throwing the exception?

Might we want to notify someone this is failing?

Reply:
I wish we had a way to buffer notice emails so we don't spam blast them over and over again. Emailing it is a good idea though.